linux_pipeline/Jenkinsfile_azure_lis_hotfix_functional

```diff
@@ -0,0 +1,344 @@
+	Command += " -CustomParameters 'DiskType=Unmanaged'"
+	println Command
+	RunPowershellCommand(Command)
+	junit "Report\\*-junit.xml"
+	archiveArtifacts '*-TestLogs.zip'
+	def CapturedVHD = readFile 'CapturedVHD.azure.env'
+	def StashName = currentVHD.replace(" ","_")
+	writeFile file: "TEST-${StashName}-CapturedVHD.azure.env", text: "${CapturedVHD}"
+	stash includes: "TEST-${StashName}-CapturedVHD.azure.env", name: "TEST-${StashName}-CapturedVHD.azure.env"
+	println("Captured VHD : ${CapturedVHD}")											
+
+}
+
+stage ("Prerequisite") {
+    node ("meta_slave") {
+        cleanWs()
+        git branch: env.GIT_BRANCH, url: env.GIT_REPO
+        stash includes: '**', name: 'LISAv2'
+        cleanWs()
+    }
+}
+
+currentStage = "Capture Image for Deploy Test"
+stage ("${currentStage}") {
+	def parellel_jobs = [:]
+	def delay = 0
+	println "DistroList ${DistroList}"
+	for (i = 0; i < total_test_images; i++)	{
+		currentTest = "CaptureVHD_${DistroList[i]}"
+		def currentDistro = DistroList[i]
+		println "currentTest ${currentTest}"
+		if(currentDistro != 'NA') {
+		    parellel_jobs ["${currentTest}"] = 
+		    {
+			    println "running parallel jobs"
+			    stage ("${currentTest}") {
+				    try
+				    {
+				        delay += 5
+						println "Sleeping ${delay} seconds..."
+						sleep "${delay}"
+						println "Running ${currentTest}..."
+						node('azure')
+						{
+							CleanWorkspace()
+							withCredentials([file(credentialsId: 'Azure_Secrets_TESTONLY_File', variable: 'Azure_Secrets_TESTONLY_File')])
+							{
+								println "Calling captureVHD_DeployTest ${currentDistro}.."
+								captureVHD_DeployTest("${currentDistro}")			
+							}
+					    }
+				    }
+				    catch (exc)
+				    {
+					    currentBuild.result = 'FAILURE'
+					    println "${currentStage}: STAGE_FAILED_EXCEPTION."
+				    }
+				    finally
+				    {
+					    //println "${it}: ${currentBuild.result}"
+				    }
+			    }
+		    }
+	    }
+	}
+	parallel parellel_jobs
+}
+
+currentStage = "DeployTests"
+stage ("${currentStage}") {
+	def parellel_jobs = [:]
+	def delay = 0
+	println "DistroList ${DistroList}"
+	
+	for (i = 0; i < total_test_images; i++)	{
+		currentTest = "InstallUpgrade_${DistroList[i]}"
+		def currentVHD = DistroList[i]
+		println "currentTest ${currentTest}"
+		if(currentVHD != 'NA') {
+		    parellel_jobs ["${currentTest}"] = 
+		    {
+				println "running parallel jobs"
+			    stage ("${currentTest}") {
+				    try
+				    {
+						println "Running Install Upgrade Scenarios..."
+						node('azure') {
+							CleanWorkspace()
+							withCredentials([file(credentialsId: 'Azure_Secrets_TESTONLY_File', variable: 'Azure_Secrets_TESTONLY_File')])
+							{
+								Prepare()
+								println "LIS-DEPLOY_SCENARIOS"
+								def Command = ".\\Run-LisaV2.ps1"
+									Command += " -XMLSecretFile ${Azure_Secrets_TESTONLY_File}"
+									Command += " -TestPlatform 'Azure'"
+									Command += " -TestLocation 'northeurope'"
```

Comment:
set up with specific region?

Reply:
Modified to get region through parameter